src/main/java/uk/gov/hmcts/divorce/systemupdate/service/task/ConditionalOrderPronouncedCoverLetterHelper.java

```diff
@@ -185,7 +186,12 @@ private Map<String, Object> templateVarsForOfflineRespondent(final CaseData case
 
         templateContent.put(DATE_OF_HEARING, dateOfHearing);
         templateContent.put(TIME_OF_HEARING, timeOfHearing);
-        templateContent.put(COURT_NAME, caseData.getConditionalOrder().getCourt().getLabel());
+        final ConditionalOrder conditionalOrder = caseData.getConditionalOrder();
+        String courtLabel = null;
```

Comment:
I dont believe we need to set to null first here. If the label is empty it will be null anyway?